docs/Developers-Guide.md

```diff
@@ -43,32 +50,43 @@ and linting python/frontend code in your local environment you need respectively
 frontend dependencies installed.
 
 For more information:
-```
+
+```python
 ! run_tests.py --help
 ```
+
 To run frontend tests in watch mode, cd to `frontend` directory and use
-```
+
+```python
 ! yarn run test --watch
 ```
+
 To run TSLint in watch mode, use
-```
+
+```python
 ! yarn run lint --watch
 ```
 
 To run a single test (there are multiple ways to do it), open a shell in the docker container:
```

Comment:
multiple ways to do it, yet only one that is provided....

what about just 

```shell
$ sudo docker exec -it $CONTAINER_ID nosetests /usr/local/src/timesketch/timesketch/lib/emojis_test.py -v
```

```

Reply:
yes this line I have not touched to the previous just wanted to give newlines before and after the `

Reply:
but I will add it